docs/reference-guides/block-api/block-bindings.md

```diff
@@ -0,0 +1,285 @@
+# Bindings
+
+The Block Bindings API lets you “bind” dynamic data to the block’s attributes, which are then reflected in the final HTML markup that is output to the browser on the front end.
+
+An example could be connecting an Image block `url` attribute to a function that returns random images from an external API.
+
+```html
+<!-- wp:image {
+	"metadata":{
+		"bindings":{
+			"url":{
+				"source":"my-plugin/get-random-images"
+			}
+		}
+	}
+} -->
+```
+
+
+## Compatible blocks and their attributes
+
+Right now, not all block attributes are compatible with block bindings. This is some ongoing effort to increase this compatibility, but for now, this is the list:
+
+| Supported Blocks    | Supported Attributes |
+| -------- | ------- |
+| Paragraph  | content    |
+| Heading | content     |
+| Image    | ID, url, title, alt    |
```

Comment:
The attribute name is different for id:

https://github.com/WordPress/gutenberg/blob/86a3bc9bb3995928b940f20a934d50a1f6f80ed3/packages/block-library/src/image/block.json#L69

```suggestion
| Image    | id, url, title, alt    |
```